src/System.Private.CoreLib/shared/System/Type.Helpers.cs

```diff
@@ -114,42 +114,42 @@ public bool IsVisible
             }
         }
 
-        public virtual Type[] FindInterfaces(TypeFilter filter, object filterCriteria)
+        public virtual Type[] FindInterfaces(TypeFilter filter, object? filterCriteria)
         {
             if (filter == null)
                 throw new ArgumentNullException(nameof(filter));
 
-            Type[] c = GetInterfaces();
+            Type?[] c = GetInterfaces();
```

Comment:
>? [](start = 16, length = 1)

In what situations can GetInterfaces return an array containing nulls?  This code obviously isn't expecting that, which suggests it's not actually possible, or so corner-case that we don't need to worry about it.

Reply:
Ah, nevermind, I see now that GetInterfaces does in fact return a Type[], but you've declared it as Type?[] presumably because we later store null into it.

---
In reply to: [278630750](https://github.com/dotnet/coreclr/pull/23489#discussion_r278630750) [](ancestors = 278630750)